### PR TITLE
Seperate the included in FO mods in a new file 

### LINKED ISSUE #335: [Suggestion] Seperate the included mods in a new Markdown file
For better organization and to reduce the size and length of the the README.md file

### PR DESCRIPTION
**Pull request by RaptaG**
Resolves #335 _(once again lol)_

### CODE REVIEW

File: INCLUDED-MODS.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+## Included mods
```
Comment (Madis0) [inline INCLUDED-MODS.md:1]: oh yeah, this needs to be # and ### needs to be ##

Comment (RaptaG) [reply INCLUDED-MODS.md:1]: Wdym, there's no `###` in the readme, and the only `#` is the title

File: INCLUDED-MODS.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+Here you can find the mod comparison table between the currently supported Minecraft versions. Some mods may have been obsoleted or superseded by better alternatives in the newer versions.
+
+The selected mods are inspired, but not limited by the [list of OptiFine alternatives](https://lambdaurora.dev/optifine_alternatives). The modpack also [improves the default settings](https://fabulously-optimized.gitbook.io/modpack/readme/changed-options).
+
+### Smooth
```
Comment (Madis0) [inline INCLUDED-MODS.md:7]: of course there is

File: INCLUDED-MODS.md (hunk 1 above)
Comment (Madis0) [reply INCLUDED-MODS.md:1]: the point is that sub-sub-headlines become sub-headlines and sub-headlines become headlines

File: INCLUDED-MODS.md (hunk 2 above)
Comment (RaptaG) [reply INCLUDED-MODS.md:7]: > of course there is

Yes.
unknown

Comment (RaptaG) [reply INCLUDED-MODS.md]: https://github.com/RaptaG/FO-greek-translations-and-more/commit/6a9c3f65692b4aa37659ac8d3206973121776362

File: INCLUDED-MODS.md (hunk 1 above)
Comment (RaptaG) [reply INCLUDED-MODS.md]: I thought it was the readme file

**Pull request merged**